v23.1/create-schedule-for-changefeed.md

```diff
@@ -0,0 +1,154 @@
+`on_previous_running` | `start` / `skip` / `wait` | Control whether the changefeed schedule should start a changefeed if the previous scheduled changefeed is still running.<br><br>`start`: Start the new changefeed anyway, even if the previous one is running.<br><br>`skip`: Skip the new changefeed and run the next changefeed based on the `RECURRING` expression.<br><br>`wait`: Wait for the previous changefeed to complete.<br><br>**Default:** `wait`
+
+## Examples
+
+Before running any of the examples in this section, it is necessary to enable the `kv.rangefeed.enabled` cluster setting. If you are working on a {{ site.data.products.serverless }} cluster, this cluster setting is enabled by default.
+
+The [Changefeed Sinks](changefeed-sinks.html) page provides detail on the available sinks for your change data messages and connection URIs. We recommend using [external connections](create-external-connection.html) to interact with external sinks. The examples in this section use an external connection URI for the changefeed sink.
+
+### Create a scheduled changefeed
+
+The following statement sets up a scheduled changefeed named `users_rides_nightly` that will send changefeed messages in `CSV` format 1 minute past midnight every night. As soon as the statement is run, the first changefeed run will execute immediately:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE SCHEDULE users_rides_nightly FOR CHANGEFEED INTO 'external://kafka-sink' WITH format=csv RECURRING '1 0 * * *' WITH SCHEDULE OPTIONS first_run=now, on_execution_failure=retry, on_previous_running=skip;
+~~~
+
+If the scheduled changefeed runs into an error, it will retry the changefeed immediately. The changefeed will skip a new changefeed run if the previous run is still running at the next scheduled time.
+
+### Create a scheduled changefeed with CDC transformations
+
+You can use CDC transformations with scheduled changefeeds to define expression syntax that selects columns and applies filters to further restrict or transform the data in your changefeed messages. When you add this expression syntax to your changefeed statement, you can only target one table. 
+
+For guidance on syntax and more example use cases, see [Change Data Capture Transformations](cdc-transformations.html).
+
+This scheduled changefeed filters for the usage of promotion codes in the [`movr`](movr.html) database and sends the changefeed messages on a daily basis:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE SCHEDULE FOR CHANGEFEED INTO 'external://kafka-sink' AS SELECT user_id, usage_count FROM movr.user_promo_codes WHERE usage_count > 1 RECURRING '@daily' WITH SCHEDULE OPTIONS first_run=now, on_execution_failure=reschedule, on_previous_running=skip;
+~~~
+
+### View scheduled changefeed details
+
+To [show](show-schedules.html) all scheduled changefeeds:
```

Comment:
I live the linked verbs for examples that introduce a new verb statement. Works well!

Reply:
Oh good!